### ISSUE TITLE
New Maintainer Wanted :-)

### ISSUE THREAD

**Issue by maelle** [cvitolo/r_hddtools#36]
(or new maintainer _team_)

If you're interested, please comment in the issue or email `info@ropensci.org`.

For more info, see
- [our guidance on Changing package maintainers](https://devguide.ropensci.org/changing-maintainers.html);
- [our Package Curation Policy](https://devguide.ropensci.org/curationpolicy.html);
- [Package Maintainer Cheatsheet](https://devdevguide.netlify.app/maintenance_cheatsheet.html);
- the blog post [What does it mean to maintain a package?](https://ropensci.org/blog/2023/02/07/what-does-it-mean-to-maintain-a-package/).

@ropensci/admin @dhugpeter

**Comment by stitam** [comment]
Hi @maelle are you still looking for a new maintainer for `hddtools`?

**Comment by maelle** [comment]
Yes, there has been no taker yet. Are you interested? 😀

**Comment by stitam** [comment]
Yes, I am interested. 😃 The domain is familiar to me and since I already maintain other data access packages in the rOpenSci ecosystem I hope this will not be a huge leap. I'd be happy to give it a try.

**Comment by maelle** [comment]
@stitam ok, thanks a ton! You're now in a team with admin access to this repository 😁

**Comment by maelle** [comment]
Re-opening as @stitam will no longer be maintaining this package (thanks anyway and thanks for your transparency, @stitam!)

**Comment by maelle** [comment]
We will archive this package as no one volunteered to take it on. #36
